style.css

```diff
@@ -1,26 +1,45 @@
-.follow-nature li {
-  position: relative;
+.follow-nature ul {
+    overflow: visible;
+}
 
+.follow-nature li {
+    position: relative;
 }
 
-a::after {
-  content: "RSS feed";
-  position: absolute;
-  background-color: #000;
-  padding: 10px 20px;
-  color: #fff;
-  border-radius: 15px;
-  top: 30px;
-  left: 50%;
-  width:auto;
-  font-family: monospace;
-  z-index: 99;
-  display: none;
-  transform: translateX(-50%);
+.follow-nature a::after {

```

Comment:
these are styles that apply to all of the `::after` pseudo elements therefore `a::after` is a generic enough selector is it applies it to them all. this prevents us duplicating these styles for each and every tooltip. you'll see below that only the things that specific to each tooltip have been set